download_protoc.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd $(dirname $0)
+
+PROTOC_VERSION="22.0"
```

Comment:
Could we pass this value in from the Makefile for clarity?

What version were we using before? I noticed the outputs are changed.

Reply:
done